app/models/enterprise.rb

```diff
@@ -34,6 +34,7 @@ class Enterprise < ApplicationRecord
   preference :shopfront_product_sorting_method, :string, default: "by_category"
   preference :invoice_order_by_supplier, :boolean, default: false
   preference :product_low_stock_display, :boolean, default: false
+  preference :white_label_logo_link, :string, default: ""
```

Comment:
Since we changed the other options to attributes, shall be do the same here as well? Otherwise it's inconsistent.

Reply:
Done, and squashed!